docs/easy-deployments.md

```diff
@@ -0,0 +1,38 @@
+# Easy Deployments
+
+There are several easy deployments instructions described in this document to help you deploy your fork of the ott-web-app to some of the popular web application hosting platforms.
+
+Currently supported platforms:
+
+- [Github Pages](#github-pages). It's a free web hosting service offered by Github. The main Github Pages advantages are that it's free of charge and easy to use.
+
+Before you follow any of ://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages
+these instructions make sure you first follow the instructions in [docs/build-from-source.md](docs/build-from-source.md).
+ 
+## Github Pages
+
+You can learn more about github pages [here](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages). 
+
+### Technical limitations
+
+Github pages is provided free of charge but it comes with some [usage limits](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits).
+In very general terms, this means that a maximum of about 20.000 **NEW** users can visit your app each month (1). If you grow faster than that, you should probably consider investing more in your hosting platform.
+
+(1) Based on 100GB Github Pages bandwidth / 5MB ott-web-app build dir size and assuming perfect browser cache by all visitors.
+
+### Usage instructions
+
+1. Make sure you can [build this project from source](./build-from-source.md). You need to install optional dependencies!
+1. Make sure you enabled Github Pages feature for `gh-pages` branch in your repository (see [Github Pages Getting Started docs](https://docs.github.com/en/pages/getting-started-with-github-pages))).
+1. Run `yarn deploy:github` and follow instructions on screen.
+
+If you need some customization option check out `yarn deploy:github --help`.
+
+### Technical documentation
+
+`yarn deploy:github` executes a simple nodejs script located in `scripts/deploy-github.js`. That script does following things in this order:
+
+- Runs `yarn build` with `SNOWPACK_PUBLIC_BASE_URL` envvar. That envvar is used to set the url location of the project. By default if SNOWPACK_PUBLIC_BASE_URL is empty the value will be based on the `git remote get-url origin` command. You can also pass your own SNOWPACK_PUBLIC_BASE_URL envvar by running `SNOWPACK_PUBLIC_BASE_URL=/my-base/ yarn deploy:github`.
+- Runs `yarn gh-pages -o origin -d build`. You can change the remote from "origin" to "myremote" by running `yarn deploy:github --github-remote=myremote`.
```

Comment:
I'm a bit confused about what changing the origin will do/ why users want to do this?

Reply:
Thanks for the review! :) I've added a bit more explanation here, does this make it clearer?

Reply:
Yep! Makes more sense